src/routes/(console)/project-[region]-[project]/overview/onboard.svelte

```diff
@@ -647,6 +707,30 @@
                 width: 250%;
             }
         }
+        .double-width-card .full-height-card {
+            height: 240px;
+            width: 100%;
+        }
+
+        @media (max-width: 1024px) {
+            .double-width-card .full-height-card {
+                height: 220px;
+            }
+            .auth-image {
+                height: 180px;
+                background-position: center bottom;
+            }
+        }
+
+        @media (max-width: 768px) {
+            .double-width-card .full-height-card {
+                height: auto;
+            }
+            .auth-image {
+                height: 160px;
+                margin-right: 0;
+            }
+        }
```

Comment:
afaik, with `scss` you can do something like -
```css
.some-class {
  /* default styles here */
  
  @media (max-width: 768px) {
    /* viewport specific styles */
   }
  
  @media (max-width: 1024px) {
    /* viewport specific styles */
   }
}
```
would be nicer to read and understand.